### ISSUE TITLE
vncpasswd: command not found [TigerVNC, XFCE4]

### ISSUE THREAD

**Issue by algeronwaterschoot** [sdarwin/Ansible-VNC#9]
Resolved by installing the `tigervnc-common` package.
https://github.com/sdarwin/Ansible-VNC/blob/129aa86825f320ef9ee7b9f50b97a6eba8555cf2/vars/Ubuntu-20.yml#L30

**Comment by sdarwin** [comment]
Hi @algeronwaterschoot,

What is the error or problem you are seeing?

--

tigervnc-standalone-server "recommends" tigervnc-common. 
When installing tigervnc-standalone-server, they are both installed:

```
The following additional packages will be installed:
  tigervnc-common
Suggested packages:
  xfonts-100dpi | xfonts-75dpi
The following NEW packages will be installed:
  tigervnc-common tigervnc-standalone-server
```

**Comment by sdarwin** [comment]
It's probably a good idea to follow your suggestion, and add tigervnc-common to the list. I will update the repo when I have a chance.

**Comment by sdarwin** [comment]
Fixed in c64f289c883f4323732ca4b27762e89e6a6b7c61
